### ISSUE TITLE
Notifications are minimally working (Android)

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#776]
It will be difficult to get notifications working perfectly, but we should get started. 

Acceptance criteria: if Quiet is connected and it receives a message, a notification should display that shows the message contents and the message sender.

**Comment by siepra** [comment]
blocked by https://github.com/TryQuiet/ZbayMobile/issues/93

**Comment by siepra** [comment]
I think we should use WorkManager and schedule tor and backend firing when the app process is not alive

**Comment by siepra** [comment]
Let's focus on displaying notifications when app is in background but main application process lives

**Comment by siepra** [comment]
This task is too general - we need to handle other things and split notifications-related work into parts

**Comment by holmesworcester** [comment]
Okay let's talk about how!

**Comment by holmesworcester** [comment]
Let's move this ticket into the sprint as a placeholder just so we're aligned on what the goal is. We can split it into parts and put those parts as higher priority.

**Comment by siepra** [comment]
Those are two parts of this task https://github.com/TryQuiet/quiet/issues/848 and https://github.com/TryQuiet/quiet/issues/857

**Comment by siepra** [comment]
I'm closing it and moving to "done" column as we changed the app behavior in this place and we have other similar tasks ready for QA
